roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -282,3 +282,16 @@
     that: ansible_os_family not in ["Flatcar Container Linux by Kinvolk"]
     msg: "download_run_once not supported for Flatcar Container Linux"
   when: download_run_once or download_force_cache
+
+- name: Verify synchronized time before install/upgrade
+  shell: "set -o pipefail && timedatectl status | grep synchronized | awk -F ': ' '{ print $2 }'"
```

Comment:
Is timedatectl actually available on all the distro's that we support? I'm not sure of that

Reply:
I checked documents of official website according to [Supported Linux Distributions
](https://github.com/kubernetes-sigs/kubespray#supported-linux-distributions)

- Flatcar Container Linux: https://docs.flatcar-linux.org/os/configuring-date-and-timezone/
- Debian: https://manpages.debian.org/buster/systemd/timedatectl.1.en.html 
- Ubuntu: http://manpages.ubuntu.com/manpages/xenial/man1/timedatectl.1.html 
- CentOS/RHEL: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/chap-configuring_the_date_and_time 
- Fedora: https://docs.fedoraproject.org/en-US/fedora/rawhide/system-administrators-guide/basic-system-configuration/Configuring_the_Date_and_Time/ 
- openSUSE: http://polarhome.com/service/man/?qf=timedatectl&tf=2&of=OpenSuSE&sf=1
- Oracle Linux: https://docs.oracle.com/en/operating-systems/oracle-linux/7/osmanage/ol7-system-datetime.html 

Seems timedatectl may available on these distributions.
What do you think?